leases/models.py

```diff
@@ -194,13 +194,30 @@ def clean(self):
                 raise ValidationError(
                     _("Cannot change the place assigned to this lease")
                 )
+
+            old_instance = WinterStorageLease.objects.get(id=self.id)
```

Comment:
you cn avoid additional DB queries by doing this:
```python
old_instance = WinterStorageLease.objects.get(id=self.id)
if old_instance.place != self.place:
    raise ValidationError(
        _("Cannot change the place assigned to this lease")
    )
self._check_application_customer(old_instance)
```
it's done same way on the `BerthLease`

Reply:
Fixed.